src/main/fc/settings.yaml

```diff
@@ -2253,6 +2253,11 @@ groups:
         default_value: 10000
         field: general.waypoint_safe_distance
         max: 65000
+      - name: nav_multi_waypoint_mission_index
+        description: "Index of mission selected from multi mission WP entry. 1 is the first WP mission in the entry. If index exceeds number of missions in the entry then no mission is loaded."
+        default_value: "1"
```

Comment:
this should be `default_value: 1`, so without the double-quotes.

Reply:
Yep I know. Various settings need updating in line with the new default settings. Already fixed my test version, just need to update affected PRs.